public/static-checkout/index.html

```diff
@@ -0,0 +1,435 @@
+                    <input id="cardcode" type="text" name="cardcode" aria-label="security code"><svg class="lock-icon" aria-hidden="true" focusable="false" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><path fill="#171c29" d="M400 224h-24v-72C376 68.2 307.8 0 224 0S72 68.2 72 152v72H48c-26.5 0-48 21.5-48 48v192c0 26.5 21.5 48 48 48h352c26.5 0 48-21.5 48-48V272c0-26.5-21.5-48-48-48zm-104 0H152v-72c0-39.7 32.3-72 72-72s72 32.3 72 72v72z"></path></svg><br>
+                  </div>
+                </div>
+                <div class="row">
+                  <div class="col-6">
+                    <label for="cardmonth"><span>Expiration Month (required)</span></label><br>
+                    <select id="cardmonth" name="cardmonth" aria-label="expiration month">
+                      <option selected>Month</option>
+                      <option>01</option>
+                      <option>02</option>
+                      <option>03</option>
+                      <option>04</option>
+                      <option>05</option>
+                      <option>06</option>
+                      <option>07</option>
+                      <option>08</option>
+                      <option>09</option>
+                      <option>10</option>
+                      <option>11</option>
+                      <option>12</option>
+                    </select><br>
+                  </div>
+                  <div class="col-6 ml">
+                  <label for="cardyear"><span>Expiration Year (required)</span></label><br>
+                    <select id="cardyear" name="cardyear" aria-label="expiration year">
+                      <option selected>Year</option>
+                      <option>2020</option>
+                      <option>2021</option>
+                      <option>2022</option>
+                      <option>2023</option>
+                      <option>2024</option>
+                      <option>2025</option>
+                      <option>2026</option>
+                      <option>2027</option>
+                      <option>2028</option>
+                      <option>2029</option>
+                      <option>2030</option>
+                      <option>2031</option>
+                      <option>2032</option>
+                      <option>2033</option>
+                      <option>2034</option>
+                      <option>2035</option>
+                    </select>
+                  </div>
+                </div>
+              </div>
+
+              <button id="place-order-button" class="submit-button">Place Order</button>
```

Comment:
It would be better/safer to make this a `<button type="submit">` and handle the form submit event instead of the click event on this button done below: `checkoutButton.addEventListener('click', function submitCybersource(event)...`. So instead:
```
<form id="checkout-form">...</form>
```
```
checkoutForm.addEventListener('submit', submitCybersource...)
```